examples/fullstack/frontend/lib/config.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+enum Environment {
+  dev,
+  prod,
+}
+
+extension enumToString on Environment {
+  String toName() {
+    return this.toString().split('.').last;
+  }
+}
+
+extension stringToEnvironment on String {
```

Comment:
I know extension types are nice, but this not ideal. We're adding this function to EVERY String, when it's only really useful is some targeted places.

Reply:
Just a nit, but really every string has *access* to the function. Wouldn't do this for a library, just a utility for the app, but not to be obtuse, I agree we should be showing best practices. Thanks.